README.md

```diff
@@ -9,49 +9,34 @@
 
 [OpenVoiceOS](https://openvoiceos.org/) is an open source platform for smart speakers and other voice-centric devices.
 
-[Mycroft](https://mycroft.ai) is a hackable, open source voice assistant by MycroftAI. OVOS-core is a
-backwards-compatible descendant of [Mycroft-core](https://github.com/MycroftAI/mycroft-core), the central component of
-Mycroft. It contains extensions and features not present upstream. All Mycroft Skills and Plugins should work normally
-with OVOS-core. Other Mycroft-based assistants are also believed, but not guaranteed, to be compatible.
+[Mycroft](https://mycroft.ai) was a hackable, open source voice assistant by the now defunct MycroftAI. OpenVoiceOS continues that work and ovos-core (this repo) is the central component.
+
+All Mycroft Skills and Plugins should work normally with OVOS-core. Other Mycroft-based assistants are also believed, but not guaranteed, to be compatible.
```

Comment:
We should call out that media skills will probably not work with OVOS-core due to OCP

Reply:
OCP works with ovos-core, and even classic mycroft ;)

Reply:
OCP does - media skills from classic Mycroft don't with OCP